### PR TITLE
Make possible to serve to the browser for Ionic2 projects

### PR DESCRIPTION
**Pull request by matrosov-nikita**
- Check logs from Ionic2 dev server to determine whether it's ready

- Increase default server timeout since loading server for Ionic2 projects takes longer

- Use 'tree-kill' to kill properly processes spawned by 'ionic serve'

There was bug with finding an open port for the notification server.It's strongly recommended to update 'ionic-app-scripts' to the 0.0.48 version otherwise you get an error: 'Error: listen EADDRINUSE 0.0.0.0:53703'

Related issues: https://github.com/Microsoft/vscode-cordova/issues/134, https://github.com/Microsoft/vscode-cordova/issues/213

### CODE REVIEW

File: src/debugger/extension.ts [hunk 1]
```diff
@@ -69,21 +70,7 @@ export function cordovaStartCommand(args: string[], cordovaRootPath: string): ch
...
-        ];
-
-        return execCommand(command, args, errorLogger).then(() => void 0);
-    } else {
-        childProcess.kill();
-
-        return Q<void>(void 0);
-    }
+export function killChildProcess(childProcess: child_process.ChildProcess): Q.Promise<void> {
+    TreeKill(childProcess.pid, 'SIGKILL');
```
Comment (MSLaguana) [inline src/debugger/extension.ts]: Is this change necessary, or just convenient? Where possible we've tried to limit the number of third party dependencies we have.

Comment (matrosov-nikita) [inline src/debugger/extension.ts]: @MSLaguana, I think this change is necessary. There are two processes spawned by serve command - [first](https://github.com/Microsoft/vscode-cordova/blob/034090b374c2ed8b00e5e1616fd374c1d8e54788/src/debugger/extension.ts#L69) is spawned by extension and [second](https://github.com/driftyco/ionic-cli/blob/master/lib/utils/npmScripts.js#L72) is spawned by ionic-cli. The second process remains in memory after debug session is being terminated.  Using `tree-kill` package is the only solution which I could find to kill that second process.

Comment (MSLaguana) [inline src/debugger/extension.ts]: I see, fair enough.
